public/stylesheets/site/2.0/18-zone-searchbrowse.css

```diff
@@ -94,8 +98,8 @@ form.filters dl {
   width: 100%;
 }
 
-.filters input[type="radio"] + label, .filters input[type="checkbox"] + label {
-  padding-left: 0.375em;
+.filters input + label {
```

Comment:
Get rid of the excessive padding between the checkboxes/radio buttons and the labels, also to buy us some room when the filters are on the narrower side.